CLAUDE.md

```
@@ -0,0 +1,141 @@
+npx eslint src/
+
+# Fix ESLint issues
+npx eslint src/ --fix
+
+# Format with Prettier
+npm run format
+```
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)